app/internal/commandsDB/databaseFunc.go

```diff
@@ -0,0 +1,115 @@
+package commandsDB
+
+import (
+	"database/sql"
+	"errors"
+	"time"
+	"todo-list-project/app/config"
+	"todo-list-project/app/internal/models"
+)
+
+// создаем задачу в БД
+func InsertInDB(task models.Remind) (uint64, error) {
+	query := "INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)"
+	res, err := config.DB.Exec(query, task.Date, task.Title, task.Comment, task.Repeat)
+	if err != nil {
+		return 0, err
+	}
+	// получаем ID созданной записи
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
```

Comment:
Ошибку лучше оборачивать, чтобы понимать, где она случилась
```suggestion
		return 0, fmt.ERrorf("error while getting last id: %w", err)
```